.github/workflows/e2e_test.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           - "e2e-test-eibc-corrupted-memo-evm"
           - "e2e-test-eibc-excessive-fee-evm"
           - "e2e-test-eibc-fulfillment-evm"
+          - "e2e-test-eibc-fulfillment-ignore-hub-to-rollapp"
```

Comment:
should add both rollapp-evm and rollapp-wasm

Reply:
added 🫶